docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+NeMo fundamentals
+=================
+
+On this page we will go a little bit deeper on how NeMo works, to give you a good foundation for using NeMo for your :ref:`desired usecase <where_next>`.
```

Comment:
Suggested revision.

On this page, we’ll delve deeper into how NeMo works, providing you with a solid foundation to effectively use NeMo for your :ref:`specific use case <where_next>`.

Reply:
I like it but could we use a word other than "delve"? These days people are saying that the word "delve" is a sign that the document was written by ChatGPT, since it likes to use that word so much :)

Reply:
Sure, good feedback. How about "we'll look into how"

Reply:
Changing to:
On this page, we’ll look into how NeMo works, providing you with a solid foundation to effectively use NeMo for you :ref:`desired usecase <where_next>`.

Reply:
We don't really use the word "desired" in technical documentation. I suggest changing it to:

 On this page, we’ll look into how NeMo works, providing you with a solid foundation to effectively use NeMo for your :ref:specific use case <where_next>.